tests/integration/targets/mso_schema_template_vrf/tasks/main.yml

```diff
@@ -146,6 +147,7 @@
     vrf: VRF2
     layer3_multicast: true
     vzany: true
+    ip_data_plane_learning: disabled
```

Comment:
Should we add test for enabled case?

Reply:
the value is enabled by default. to validate added an assert in name: Verify vrf3_nm